pyccel/ast/numpyext.py

```diff
@@ -1960,19 +1960,26 @@ def is_elemental(self):
 
 class NumpyAmax(NumpyUfuncUnary):
```

Comment:
It looks like this class inherits from the wrong super class. All your changes look great but they mean that `NumpyUfuncUnary` is no longer used. Looking at NumPy's definition of a unary function max doesn't seem to match anyway. Please can you change the super class to `PyccelInternalFunction`

Reply:
I have changed it please check and tell me if it's ok or not